src/base-element.js

```diff
@@ -134,6 +135,14 @@ export class BaseElement {
     this.resources_ = resourcesFor(this.getWin());
   }
 
+  /**
+  * Returns the element priority. A value from 0 to 10.
```

Comment:
`getPriority` sounds better to me. This is the priority of loading elements (`layoutCallback`). The lower the number, the higher the priority.